tests/integration/test_notebooks.py

```diff
@@ -88,18 +88,18 @@ def test_criteo(asv_db, bench_info, tmpdir):
         print("Tensorflow not installed, skipping " + notebook)
 
 
-def test_rossman(asv_db, bench_info, tmpdir):
+def test_rossman(asv_db, bench_info, tmpdir, devices):
     data_path = os.path.join(DATA_DIR, "rossman/data")
     input_path = os.path.join(DATA_DIR, "rossman/input")
     output_path = os.path.join(DATA_DIR, "rossman/output")
 
     # Run Donwload & Convert for all
     notebook = os.path.join(dirname(TEST_PATH), ROSSMAN_DIR, "01-Download-Convert.ipynb")
-    out = _run_notebook(tmpdir, notebook, data_path, input_path, gpu_id="0", clean_up=False)
+    out = _run_notebook(tmpdir, notebook, data_path, input_path, gpu_id=devices, clean_up=False)
```

Comment:
Does this work correctly... doesnt the converted rossman data end up in input path?  Then does step 2, the nvtabular preprocessing, run on the original data (i.e. datapath) instead of using the converted data in the input_path as input?

Reply:
It works correctly because the notebooks is using the output_dir and not the input_dir to get the date
```
DATA_DIR = os.environ.get(
    "OUTPUT_DATA_DIR", os.path.expanduser("~/nvt-examples/data/")
)
```

I didn't want to modify the notebook just in case someone is relying in that import and gets affected.